Dockerfile-runtime

```diff
@@ -57,22 +57,16 @@ RUN \
   . h2o4gpu_env/bin/activate && \
   pip install --no-cache-dir -r requirements.txt
 
-ADD src/interface_py/dist/ /dist
+#Below will get the latest stable wheel file instead.
 RUN \
   . h2o4gpu_env/bin/activate && \
-  pip install --no-cache-dir /dist/*.whl
-
-#Below will get the latest stable wheel file instead.
-#RUN \
-#  . h2o4gpu_env/bin/activate && \
-#  pip install --no-cache-dir https://s3.amazonaws.com/artifacts.h2o.ai/releases/stable/ai/h2o/h2o4gpu/0.0.4/h2o4gpu-0.0.4-py36-none-any.whl
+  pip install --no-cache-dir https://s3.amazonaws.com/artifacts.h2o.ai/releases/bleeding-edge/ai/h2o/h2o4gpu/0.0.4/h2o4gpu-0.0.4-py36-none-any.whl
```

Comment:
Bleeding edge of cuda 8 build for python 3.6

Reply:
The same as above, make it more parameterized.